src/lib/MsDyn.Contrib.CloneFieldDefinitions/CloneFieldDefinitionsPlugin.cs

```diff
@@ -15,7 +15,7 @@ namespace MsDyn.Contrib.CloneFieldDefinitions
XW6Pl+enEEz6Xx5mHSHbZnsBsNXQ04rXcaGgeGRQy6SyMyis2sKtYaGBwcwNY/bcO2bVuzFpkgypgKwE6ebbcAd73yGLzRH5VbmOjdXa2DK8cpe36Z4kdPOumkBw4nXB12QAeAhYsWfihhftDV5JQECre4pcaitraiGyYSBF2mwOvp6sYNf38D2tvXGZWdiN5FJJ3s0ydZfZeI0iCr6ROcduqpOPnkk9DU3BLO1Uce9j6Vzu7c09SHjSPXA6M1hte11lDCIV+bK5UbRZXArlW3/yU1uuyjFwG/CPSpTtFX6QN6gFN6T8PwUqvPyN264nhfqxfB7Nu3D1s6O7F169ZMw+6iuBlEMdE6JthsCu5650EablJ1ZnCQkosBFIpS+IBF95bMziNOycOAPnz61KkPHW6YOiyBDgCLFi66ghP6BXOifGuMTLptASjUbokcSY1EMSIDKJXKuOGG/4UXXnjBlAVGb2/aauQY+jwDf6A/kyZNwpQpUzBmzFi/McaslSJIr7u8o+2+xr3YPGIjBkaWMbpxzAEAXlsgcyjJuNAU0n9nV2knGnobMGXPKWgutfi6W5Q3sgRQFuNmjHTnKzvR0dGBbdu25Zj0LJqawztiwm1LjQ3p5tL1uHduZsa9ui07RNik3YrY1+0x8MW4qe+ZE5RSmki976yzznr0cMTTYQv0LI1fdHGS8MPMPDwcQ5US2XxN7nrvFCrhbCpeLpfxD//7K1j19NM+kgsSLhGiGlf7sz0AsvcmIpdhBLU+EcaOHYvJkydj4sSJBQsnEYE937JL66rY2rIFO1tfQXNLC1qSlhz19moIub+2Mq5WTb63shd79u3B2D1tOGHvRCSVJOp1K1+DKx28rNutnqbYtGkTNm/ejN1dXW5stAjotuZ2Tqvmd+0UbRbAEQvva3AhgomIN5IurgrRgSDMMoMWGvURqffPmDlz2eGKpcMa6ACwaNHCc4mSh5KEJ8WS16TOEmUctuCYcsIaFvcxM7QG/vGrX8ETTzyRa8vt74/LAsRr1frT0NCAiRMn4sQTT8w2yHiJXW7nu6zrlQpNLyv1Fexu3IVdTTswOGwQI4aNRCM31gD3/oQzr52CiyN4uVpCd6kLSX89xpXaMKbchmSwrvA1lO1123fX4fQbFLBjxw5sWL8eGzduRFpNIzLN/2FiH5GD1Jz8Aky7TSVqkzkhTCygsZGd81LWWN+ekXqh6s3U+R1E9KHzzz//mcMZR4c90LM0fuGpnCQPsJHLZmSZrcFtxKX8lBpHzDyFab5SCt/5zrdx/wMPBDwAsW+pZak8hUy/IOYSNsSf0dH7Q4DNdF722FGjRuH4E47HcROOQ2NjYxjljcuN3zsDMX+vCpZWADqporehF7sadmFvwx6gQaOpoRmtyYiav9TXspKpt9KL/oF9UGVC00AzxpTHonVwBLjCRZAO0nAJ5CB6m6+9Pb1Yv34d1m9Yj12v7ArBbaJzHuxs/t+GgymuJnfg5oBMs/e5VLum1FUAX6ncoRCm8U4s8xyR+sgFF1yw/nDH0BEBdBPZhzMn9zPz5a7ujlJzL2FNimts+z3yix+IGfPuvhv//sM7szotYO45p6sPonmc7tvHkqj1TalBbObvmdHW1obx48dj3Lg2w9iLutuaV2ottPUhmRdnBPHG2mrdIPbxPuxL9mFf3V6UkhKqXEHKGmbPJHQKqIpCXbUODZVGNFea0VxpQWN1GOqqddApfJotCQX3Y2Xtr2yNsZYBO0S4Id729PWhvb0d7eva0dmxJUu12bez5Fdi45zLUvAio7tMxTkQv4RLFrw4hoM+uGTTKR+9KVq4QBFJl42hLrzk0ktnHyn4OWKAbq+lS5d+n5n/LraYCoZTLABlJE8i/zgBzCRJ8Ogjj+K2/3NbppVmyqfz0aHilXXm/c39WS1PIEMe2oMg0wJkM/CckMkgCGPGjEFbWxvGjB6NpubmaEWV9yYP7Y4F4HS0rRayJDAYVNGv2nrqI3PPVblPggzF+brafcs9LB+6tQJ2bN+O9vZ2rF27Fp2dnVGtzUbAwkHUzg4AT6iRZcipGPQUzY6zrMtzM+ZWyx4uSPTDKhwJYSCyB8iS4AezZ8++7kjCzREHdABYsnTpjQnzNzhn/phP230vnAqcZkLQP/nkk/ja176GgXLZ+c0RkxmKEbW/fD8qrtXlYUFEkfhGHDTkn9vc3IKxY0dj5MhRGN7aivr6OkDL1dJ26YUk+LQDO4KU36PdLRDW0jTfp9nBgaIB7Xz2D1zLK/GXvr17sGnjRrS3t2Pdunbs3LkrSMnjqJ0HbwY0x6iT/f/vAcqyFcbRFBpR8Ho1ozSFxFsgpCmQtbqDJLv/i+973/tuPdIwc0QCPYvsSz7AnMxj5uF+sEVo1GOxSzC/nt2O9e7MjNWrV+OmL30JXd3dr6G1lkXoQF1HBaDmiMQjm+abyJ/47zETmlta0NraiuaWFjQ3N6Np2DAkzCaIxlScDhdiaIUQ14b7rgHibG+4bIFpQaqp3CfFGiC+8PsXsHLl03jppU3Y/vJ2UMJunRWxyv6/uKgsiTUDaAHMTG/OIgKbCB2Bm+IDwra+WA6lcFSTRxNnBeAPzByNKYUi2FS/j0h9/CMf+ejPj0S88JEK9B//+Cd/vOqqqx9VwEwonBiUh9GKJq11ATcN0+O2dWh2/4QJ4/G2WbPw618/g57uHh+9lPUgyygzbdbCqBrzXhp2pNXmuPJOn/fK7rR9fasNrwxWsK9/L3p6erB7105s3/EKurq6sG/fPpTLZVQqmSMqsQJx4nT3qmA3nZLLMCwPoFTBRlFpepHt8+Y6RkN9A5qamtDS0oKRo0Zh7NixGDV6FM4880y0DG/BqlWr0NPdE46oauFGo0MnWSuG8+aK0W3pNCuWI9jXtdtU3BYVmAUJwWv6xQppvFBBrCj2+8/8c9x20+x9V2mt//bKK6966kjFyxEb0eW1bNmy7yfGmsoZSHLi5KrSh87V5QWpvPzT1dWFm266Ca
```

Comment:
Please leave the name as is, the version number will update during the next release, so we don't need this.